pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go

```diff
@@ -132,12 +135,12 @@ func (defaultauthZ *DefaultAuthZImpl) Handle(command utils.CommandEnum, cluster
 		return defaultauthZ.nextHandler(command, cluster, w, r, swarmHandler)
 
 	case utils.NETWORK_CONNECT, utils.NETWORK_DISCONNECT:
-		if err := ConnectDisconnect(cluster, r); err != nil {
+		if err := ConnectDisconnect(cluster, r, w); err != nil {
 			return err
```

Comment:
Maybe instead of adding response to the call and do the error from the connect/disconent function just return an error  with the http etc.. from here similar to what you do in the other "cases"